src/metalnx-web/src/main/java/com/emc/metalnx/controller/TemplateController.java

```diff
@@ -301,8 +306,24 @@ public String modifyTemplate(final Model model, @ModelAttribute final MetadataTe
 				throw new Exception("Cannot modify a non-existent template");
 			}
 			DataGridUser loggedUser = loggedUserUtils.getLoggedDataGridUser();
-			if (!template.getOwner().equalsIgnoreCase(loggedUser.getUsername())) {
-				throw new Exception("Cannot modify a template beloging to another user");
+
+			// throw error if user is not the template owner, rodsadmin or groupadmin
+ 			if (!(template.getOwner().equalsIgnoreCase(loggedUser.getUsername())) && loggedUser.isAdmin() == false && loggedUser.isGroupAdmin() == false) {
+				throw new Exception("Cannot modify a template belonging to another user");
+			}
+			// throw error if logged user is not the rodsadmin of the template owner's group	
```

Comment:
in this comment...    rodsadmin -> groupadmin

Reply:
yep, thanks for catching that